### PR TITLE
Mobile portfolio home page

### PR DESCRIPTION
**Pull request by sheldonwafula**
In this pull request:
I have created this project using HTML and CSS.
I have added a descriptive readme file.
I have corrected all linter errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+<body>
+    <section class="landing-page">
+    <nav class="nav-bar">
+        <div class="logo">
+<a href="#"> My logo</a>
+        </div>
+        <div class="nav-icon">
+<img src="images/Union.png">
+        </div>
+    </nav>
```
Comment (levy002) [inline index.html:19]: - [ ] Kindly make sure to wrap the navigation container into the `header` tag. Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info..

File: index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    </p>
+<a href="#" >LET’S CONNECT</a>
+<ul class="social-media">
+    <li><a href="#"><img src="images/vector.svg"></a></li>
+    <li><a href="#"><img src="images/Linkedin icon.svg"></a></li>
+    <li><a href="#"><img src="images/vec.svg"></a></li>
+    <li><a href="#"><img src="images/github.svg"></a></li>
+    <li><a href="#"><img src="images/slack.svg"></a></li>
+</ul>
+</div>
```
Comment (levy002) [inline index.html:36]: - [ ] Kindly make sure to wrap the headline container into the `section` tag. It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @sheldonwafula   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Professional PR description :heavy_check_mark: 
- Good UI :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+<ul class="social-media">
+    <li><a href="#"><img src="images/vector.svg"></a></li>
+    <li><a href="#"><img src="images/Linkedin icon.svg"></a></li>
+    <li><a href="#"><img src="images/vec.svg"></a></li>
+    <li><a href="#"><img src="images/github.svg"></a></li>
+    <li><a href="#"><img src="images/slack.svg"></a></li>
+</ul>
+
+</section>
+    </section>
```
Comment (binhussen) [inline index.html:40]: -  Great Job for correctly adding the `header` tag around the toolbar and the `section` tag around the headline section,👍👍 however you have an extra `section` tag as the previous code reviewer suggested, please remove it for semantic reasons 👍
- check the below  👇 structure
```
<body>
   <header>
      ......
   </header>
   <section>
     .....
   <section>
</body>
```

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @sheldonwafula , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @sheldonwafula     👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job so implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉 

## Highlights
✔️professional PR👍 
✔️professional README👍 
✔️No Linter errors👍 

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**